ADRs/RH/SIG-SRE/ADR-00001 Document architecture decisions using the Architecture Decision Records approach and define their templates.md

```diff
@@ -0,0 +1,98 @@
+# ADR-00001 Document architecture decisions using the Architecture Decision Records approach and define their templates
+
+Authors: Jeremy Eder
+
+
+## Status
+
+Draft
+
+
+## Problem Statement
+
+We need to capture our architecture decisions for all of Red Hat’s SIG-SRE and consumers. This capture should make it easy for people to find a given decision and refer to it. The decision context should be captured so we can question that decision when changes arise.
+
+
+## Goals
+
+* Capture key decisions
+* Let the team rely on these decisions
+* Capture decision context so we can revisit when context changes
+* Be easily discoverable
+
+
+## Non-goals
+
+* Does not aim at achieving perfection from the get go
+
+
+## Current Architecture
+
+* None
+
+
+## Proposed Architecture
+
+* Embrace a well formed template for each Architecture Decision
+    * [[Template] Architecture Decision Record](ADR-00000%20Template.md)
+* Store each Architecture Decision Record (ADR) in [Operate First’s SRE repository](https://github.com/operate-first/sre/tree/main/ADRs/RH/SIG-SRE).
+* Each Architecture Decision Record is named “ADR-nnnnn Some title”
+    * nnnnn is a quasi monotonic number assigned at ADR creation time
+* ADRs are immutable once their status moves to accepted
+* ADRs can supersede preceding ADRs: this is how a decision is changed.
+* No ADR once moved past Draft can be ‘deleted’ or ‘removed’.
+* Drafts can be developed as pull requests or using gdocs. However once they reach Proposed, they must become PRs against the ADR repository.
```

Comment:
Here's the part that makes it clear Drafts can be developed outside of github.

Reply:
:+1: